flink/src/test/java/org/apache/iceberg/flink/SimpleDataUtil.java

```diff
@@ -259,4 +265,37 @@ public static StructLikeSet actualRowSet(Table table, Long snapshotId, String...
 
     return dataFiles;
   }
+
+  public static Map<Long, List<DataFile>> snapshotToDataFiles(
+      Table table)
+      throws IOException {
+    table.refresh();
+    Map<Long, List<DataFile>> res = Maps.newHashMap();
+    List<ManifestFile> manifestFiles = table.currentSnapshot().allManifests();
```

Comment:
since we are only looking for `DataFile`, we probably should call `dataManifests` (not `allManifests`) here

Reply:
Done